weblate/formats/exporters.py

```diff
@@ -39,18 +40,12 @@
 from weblate.formats.ttkit import TTKitFormat
 from weblate.trans.util import split_plural, xliff_string_to_rich
 from weblate.utils.site import get_site_url
+from weblate.formats.models import EXPORTERS
```

Comment:
Probably this import should not be here and rather related code moved to models.py?

Reply:
Okay. EXPORTERS is used in two methods in this file. So I will move those two methods to models.py (and also change wherever the methods are called, which is like 2-3 places).

Reply:
I have fixed this with 28ba7f1 .